### PR TITLE
feat(doc): improve cache policy documentation

### PR DESCRIPTION
**Pull request by benjaminParisel**

### CODE REVIEW

**Comment by benjaminParisel** [comment]
retest this please

**Comment by benjaminParisel** [comment]
restest this please

**Comment by educhastenier** [comment]
retest this please

File: md/cache-configuration-and-policy.md [hunk 1]
```diff
@@ -24,7 +24,8 @@ By default cache is disable for the web server embedded by Bonita studio, as it
 But you can decide to activate cache, to be closer to the production display time. To do this, 
 see [Cache configuration](bonita-bpm-studio-installation.md#enable_cache) documentation. 
 
-<a id="portal_and_mobile"> 
+<a id="portal_and_mobile">
```
Comment (educhastenier) [inline md/cache-configuration-and-policy.md]: Remove this change with single `/>` at the end

**Review by educhastenier — CHANGES_REQUESTED** [review]

**Comment by educhastenier** [comment]
retest this please

**Review by educhastenier — APPROVED** [review]

**Comment by educhastenier** [comment]
Navigation works!

**Pull request merged**